File: pset6/mario/mario.py
from cs50 import get_int

def main():
    level = 1
    height = get_positive_int()
    
    while True: 
       for i in range(height - level):
           print(" ", end = "")
       print_hashes(level)   
       print("  ", end = "")
       print_hashes(level)
       print()
       level += 1
       if level > height:
           break

def get_positive_int():
    while True:
        n = get_int("Height (1-8): ")
        if n > 0 and n < 9:
            break
    return n

def print_hashes(level):
    i = 0
    for i in range(level):
        print("#", end = "")
        
main()        

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void print_hashes(int level);
int get_height(string prompt);

int main(void){
    
	int height, level=1, i;
    
    height=get_height("Height (1-8): ");

	do{
		for(i=0; i<(height-level); i++)
			printf(" ");
		print_hashes(level);
		printf("  ");
		print_hashes(level);
		putchar('\n');
	}while(level++<height);

	return 0;
}

void print_hashes(int level){
	int j=0;

	while(j++<level)
		printf("#");
}

int get_height(string prompt)
{
    int n;
    
    do
    {
        n = get_int("%s", prompt);
    }
    while (n < 1 || n>8);
    
    return n;
}